api/v1alpha1/dspipeline_types.go

```diff
@@ -331,6 +331,8 @@ type WorkflowController struct {
 	Image         string `json:"image,omitempty"`
 	ArgoExecImage string `json:"argoExecImage,omitempty"`
 	CustomConfig  string `json:"customConfig,omitempty"`
+	// Specify custom Pod resource requirements for this component.
+	Resources *ResourceRequirements `json:"resources,omitempty"`
```

Comment:
Can you also update: https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/samples/v2/dspa-all-fields/dspa_all_fields.yaml#L11 

looks like we never added the workflowcontroller section, could you add that and also add the other fields?

Reply:
added